src/components/PickupBoxForm/PickupBoxForm.jsx

```diff
@@ -11,6 +11,7 @@ import {
   uploadBoxPhoto,
```

Comment:
"Support for jpeg, jpg, png" should be centered.

Reply:
done